__tests__/app.test.js

```diff
@@ -26,4 +27,21 @@ describe('app.js', () => {
             })
         })
     })
+
+    describe('GET /api', () => {
+        test('200: responds with an object of endpoints', () => {
+            return request(app).get('/api')
+            .expect(200)
+            .then(({body}) => {
+                
+                const file =  fs.readFile('./endpoints.json', 'utf-8')
+                return Promise.all([file, body])
```

Comment:
You could just import the endpoints.json file into here rather than having to use fs.